### PR TITLE
ER-1266 Hides filters from employer dashboard

### PR DESCRIPTION
**Pull request by dev4valtech**

### CODE REVIEW

File: src/Employer/Employer.Web/Views/Dashboard/Dashboard.cshtml [hunk 1]
```diff
@@ -20,27 +20,19 @@
...
-        <h1 class="govuk-heading-l govuk-!-margin-bottom-6">Your vacancies</h1>
+        <h1 class="govuk-heading-l govuk-!-margin-bottom-6">Vacancies</h1>
         <div asp-show="Model.HasInfo" class="info-summary govuk-!-margin-bottom-0">
             <p>@Model.InfoMessage</p>
         </div>
     </div>
 </div>
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-full">
-        <p class="govuk-body">
```
Comment (shomavg) [inline src/Employer/Employer.Web/Views/Dashboard/Dashboard.cshtml]: Why is this bit of information being removed?

File: src/Employer/Employer.Web/Views/Dashboard/Dashboard.cshtml [hunk 2]
```diff
@@ -20,27 +20,19 @@
         <div asp-show="Model.HasWarning" class="warning-summary govuk-!-margin-bottom-0">
             <p>@Model.WarningMessage</p>
         </div>
-        <h1 class="govuk-heading-l govuk-!-margin-bottom-6">Your vacancies</h1>
+        <h1 class="govuk-heading-l govuk-!-margin-bottom-6">Vacancies</h1>
```
Comment (shomavg) [inline src/Employer/Employer.Web/Views/Dashboard/Dashboard.cshtml]: Why is Your removed?

File: src/Employer/Employer.Web/Views/Dashboard/Dashboard.cshtml [hunk 3]
```diff
@@ -20,27 +20,19 @@
...
-            You can clone vacancies once they've been submitted.
-            Vacancies will be approved or rejected within 24 hours.
-        </p>
         <a asp-route="@RouteNames.CreateVacancyOptions_Get" class="govuk-button govuk-!-margin-bottom-6" esfa-automation="create-vacancy">Create vacancy</a>
     </div>
-    <br>
 </div>
 
-<div class="govuk-grid-row">
+<div asp-hide="true" class="govuk-grid-row">
```
Comment (sachpatel) [inline src/Employer/Employer.Web/Views/Dashboard/Dashboard.cshtml]: If we don't want it then surely it should be removed, `asp-hide="true"` is pointless. Is this specific for the dashboard on employer recruit?

Comment (dev4valtech) [reply src/Employer/Employer.Web/Views/Dashboard/Dashboard.cshtml]: I did offer the options to Jithesh, he opted to just hide it for now as we may eventually do this anyway.

Comment (sachpatel) [reply src/Employer/Employer.Web/Views/Dashboard/Dashboard.cshtml]: 👍

File: src/Employer/Employer.Web/Views/Dashboard/Dashboard.cshtml (hunk 2 above)
Comment (dev4valtech) [reply src/Employer/Employer.Web/Views/Dashboard/Dashboard.cshtml]: That is how prototype is designed.

**Pull request merged**